lib/dfu.js

```diff
@@ -108,12 +109,16 @@ var that = module.exports = {
 		];
 
 		that.checkBinaryAlignment("-D " + binaryPath);
-		return utilities.deferredSpawnProcess("dfu-util", args);
+		return utilities.deferredChildProcess(that.getCommand() + ' ' + args.join(' '));
```

Comment:
@endorama, is there some reason this need to be `deferredChildProcess()` as it broke the verbose display for DFU.